src/components/atomic/dynamic-form/VRadios.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+    <div>
+        <v-form-label v-bind="labelProps">
+            <div v-for="radio in radios">
+                <input
+                  type="radio"
+                  :id="radio.id"
+                  :value="radio.id"
+                  :checked="optionIsChecked(radio)"
+                  :name="name"
+                  v-model="dynamicValue"
+                  class="form-checkbox px-4 py-2 mt-2 h-5 w-5">
+                <label class="ml-2" :for="radio.id">{{radio.text}}</label>
```

Comment:
I think this might cause a conflict by having too many form labels. Also you should use v-text ideally here as well.